docs/esps/scaleway.rst

```diff
@@ -0,0 +1,215 @@
+-----------------
+
+To use Scaleway features not directly supported by Anymail, you can
+set a message's :attr:`~anymail.message.AnymailMessage.esp_extra` to
+a `dict` that will be merged into the json sent to Scaleway's
+`email API`_.
+
+Example:
+
+    .. code-block:: python
+
+        message.esp_extra = {
+            'HypotheticalFutureScalewayParam': '2024',  # merged into send params
+        }
+
+
+(You can also set `"esp_extra"` in Anymail's
+:ref:`global send defaults <send-defaults>` to apply it to all
+messages.)
+
+.. _email API: https://www.scaleway.com/en/developers/api/transactional-email/
+
+.. _scaleway-features:
+
+Supported features
+------------------
+
+Anymail supports these |Scaleway features| through the following |Anymail features|:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 25 25 50
+
+    * - |Anymail feature|
+      - |Scaleway feature|
+      - Notes
+
+    * - :attr:`~anymail.message.AnymailMessage.attachments`
+      - ``attachments``
+      - Binary attachments
+
+    * - :attr:`~anymail.message.AnymailMessage.bcc`
+      - ``bcc``
+      - Blind carbon copy recipients
+
+    * - :attr:`~anymail.message.AnymailMessage.cc`
+      - ``cc``
+      - Carbon copy recipients
+
+    * - :attr:`~anymail.message.AnymailMessage.extra_headers`
+      - ``additional_headers``
+      - Custom email headers
+
+    * - :attr:`~anymail.message.AnymailMessage.from_email`
+      - ``from``
+      - Sender email and name
+
+    * - :attr:`~anymail.message.AnymailMessage.reply_to`
+      - ``additional_headers``
+      - Reply-To header
+
+    * - :attr:`~anymail.message.AnymailMessage.subject`
+      - ``subject``
+      - Email subject line
+
+    * - :attr:`~anymail.message.AnymailMessage.to`
+      - ``to``
+      - Primary recipients
+
+
+Limitations and quirks
+----------------------
+
+Scaleway does not support a few features offered by some other ESPs.
+For a complete list of technical limitations, refer to the `Scaleway Transactional Email API documentation`_.
+
+Anymail normally raises an :exc:`~anymail.exceptions.AnymailUnsupportedFeature`
+error when you try to send a message using features that Scaleway doesn't support.
+You can tell Anymail to suppress these errors and send the messages anyway --
+see :ref:`unsupported-features`.
+
+**Attachment limitations**
+  Scaleway limits attachment types to a predefined list (e.g., common document, image, and text formats).
+  Attachment size is limited to 2MB.
+
+**Minimum content length**
+  The subject, body, and HTML content of your emails must each have at least 10 characters.
+
+**No delayed sending**
+  Scaleway does not support :attr:`~anymail.message.AnymailMessage.send_at`.
+
+**No tags support**
+  Scaleway does not support :attr:`~anymail.message.AnymailMessage.tags`.
+
+**No metadata support**
+  Scaleway does not support :attr:`~anymail.message.AnymailMessage.metadata` or
+  :attr:`~anymail.message.AnymailMessage.merge_metadata` features.
```

Comment:
We *could* support metadata and tags by sending them in custom message headers. The message headers seem to be included in the webhook payload, so could be retrieved by Anymail's tracking webhook.

(We do this for Resend. If you're interested, take a look at anymail/backends/resend.py.)